source/agora/test/Flash.d

```diff
@@ -277,6 +279,29 @@ public class TestFlashNode : ThinFlashNode, TestFlashAPI
     {
         while (1)
         {
+            if (LOG_CHANNEL)
+            {
+                writefln("chan_id %s found: %s", chan_id, !!(chan_id in this.channel_updates));
+                if (auto updates = chan_id in this.channel_updates)
+                {
+                    writefln("chan_id %s update dir %s found %s", chan_id, dir, !!(dir in *updates));
+                    if (auto update = dir in *updates)
+                    {
+                        writefln("chan_id %s update_idx %s == channel idx %s: %s", chan_id, update_idx,
+                            update.update_idx, update_idx == update.update_idx);
+                    }
+                    else
+                    {
+                        writefln("cannot find direction %s", dir);
+                    }
+                }
+                else
+                {
+                    log.info("Can't find chan ID {}. map: {}",
+                        chan_id, cast(void*)&this.channel_updates);
+                }
+            }
+
```

Comment:
This looks like debugging leftover ?

Reply:
Oops, I thought they were all logs. Will take this one out.